src/components/SearchUsers/SearchUsers.js

```diff
@@ -25,9 +25,12 @@ const SearchUsers = ({
   const { pathname } = useLocation();
   const [page, setPage] = useState(1);
 
-  const { totalPages } = users.data.pageViewModel;
+  const {
+    items,
+    pageViewModel: { totalPages, pageNumber },
+  } = users;
```

Comment:
Why not get these props in mapStateToProps? 
Also create wrapper for this connect